src/config.rs

```diff
@@ -355,6 +356,12 @@ pub struct Pool {
 }
 
 impl Pool {
+    pub fn hash_value(&self) -> u64 {
```

Comment:
I think you can just `#[derive(Hash)]` on the struct.

Reply:
What is the method name that this generates?